docs/ContribOperators.md

```diff
@@ -404,6 +404,8 @@ This version of the operator has been available since version 1 of the 'com.micr
 <dd>no repeat ngrams size</dd>
 <dt><tt>pad_token_id</tt> : int (required)</dt>
 <dd>The id of the padding token</dd>
+<dt><tt>vocab_size</tt> : int</dt>
+<dd>Size of the vocabulary.If not provided, it will be inferred from the decoder subgraph's output shape</dd>
```

Comment:
Do we need also update the shape inference for BeamSearch? One is for the operator def, and another is in symbolic shape inference.